src/Maker/MakeController.php

```diff
@@ -55,6 +56,7 @@ public function generate(InputInterface $input, ConsoleStyle $io, Generator $gen
             $controllerClassNameDetails->getFullName(),
             'controller/Controller.tpl.php',
             [
+                'parent_class_name' => class_exists(AbstractController::class) ? 'AbstractController' : 'Controller',
```

Comment:
AbstractController exists also in 3.4, so the check doesn't achieve what the title of this PR tells

Reply:
You are right 😭  Should we revert to the original code then?

Reply:
What's wrong with using `AbstractController` on 3.4 code?

Reply:
That it doesn't have the `->getParameter()` shortcut ... so it's not really equivalent to `Controller`. In 4.1 it is.

Reply:
Ok, change the check to `\method_exists(AbstractController::class, 'getParameter`) then?